plumbing/revlist/revlist.go

```diff
@@ -11,30 +12,56 @@ import (
 )
 
 // Objects applies a complementary set. It gets all the hashes from all
-// the reachable objects from the given commits. Ignore param are object hashes
+// the reachable objects from the given objects. Ignore param are object hashes
 // that we want to ignore on the result. It is a list because is
```

Comment:
`It is a list because is easier to interact with other porcelain elements, but internally it is converted to a map.`

This comment is about the internal implementation and users do not care too much about this. Could you just remove this part?